index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <title>Awesome Books</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet" crossorigin="anonymous" />
+</head>
+    <!-- Navigation Bar -->
+    <header>
+        <nav class="nav-bar">
+                <a class="brand" href="./index.html">Awesome Books</a>
+                <ul class="nav-menu">
+                   <li class="nav-item"><a class="nav-link booklist-2" href="#list">Booklist</a></li>
+                    <li class="nav-item"><a class="nav-link form-section" href="#form">Add a Book</a></li>
+                    <li class="nav-item"><a class="nav-link contact-section" href="#contact">Contact</a></li>
+                </ul>
+        </nav>
+    </header>
```

Comment:
Great implementation of dynamic display of elements, including the add book page. 🔥 However there seem to be a bug that comes along with the implementation. 

- [ ] Kindly make sure that the `Add Book` navigation link in your `Navbar` does not also disappear when you are hiding the book form. 

### Please check below ⬇️ 
![missing add book](https://github.com/katarighe/awesome-books-es6/assets/117144028/5353ccbd-dddb-4e81-8ccf-0b99e6e039fd)